Modules/Scripted/Home/HomeLib/segmentation_model.py

```diff
@@ -117,21 +118,24 @@ def run_inference(self, img):
       img_pil.save(input_file_path)
 
       # Run monai-deploy
+      monai_deploy_path = shutil.which("monai-deploy")
       if self.model_source == self.ModelSource.USE_LOCAL_ZIP:
         deploy_app_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "deploy_app.py")
-        process_image_command = ["monai-deploy", "exec", f"{deploy_app_path}",
+        process_image_command = [f"{monai_deploy_path}",
+                                 "exec", f"{deploy_app_path}",
                                  "-m", f"{self.save_zip_path}",
                                  "-i", f"{input_dir_path}",
                                  "-o", f"{output_dir_path}"]
```

Comment:
```suggestion
        process_image_command = [monai_deploy_path,
                                 "exec", deploy_app_path,
                                 "-m", self.save_zip_path,
                                 "-i", input_dir_path,
                                 "-o", output_dir_path]
```

Reply:
Can do.  It seemed "proper" to me to explicitly convert from `os.path` to `str`, which are conceptually different, but this suggestion is definitely cleaner.